FV3/gfsphysics/GFS_layer/GFS_physics_driver.F90

```diff
@@ -2046,6 +2046,18 @@ subroutine GFS_physics_driver                                 &
 
 !     if (lprnt) write(0,*) 'tisfc=',Sfcprop%tisfc(ipr),'tice=',tice(ipr),' kdt=',kdt
 
+! force SST to be equal to dynamical core surface temperature, which will be the same as
+! analysis SST when nudging is active
+      if (Model%use_analysis_sst) then
```

Comment:
is this a namelist flag? if not, where is 'use_analysis_sst' set?

Reply:
Yes that's right, it's a namelist parameter I've added to `gfs_physiscs_nml`.